roles/openshift_logging_kibana/tasks/generate_route.yaml

```diff
@@ -0,0 +1,54 @@
+---
+# create routes
+- set_fact:
+    kibana_name: "{{ 'logging-kibana' ~ ( (openshift_logging_kibana_ops_deployment | default(false) | bool) | ternary('-ops', '')) }}"
+
+# TODO: set up these certs differently?
+- set_fact:
+    kibana_key: "{{ lookup('file', openshift_logging_kibana_key) | b64encode }}"
+  when: openshift_logging_kibana_key | trim | length > 0
+  changed_when: false
+
+- set_fact:
+    kibana_cert: "{{ lookup('file', openshift_logging_kibana_cert) | b64encode }}"
+  when: openshift_logging_kibana_cert | trim | length > 0
+  changed_when: false
+
+- set_fact:
+    kibana_ca: "{{ lookup('file', openshift_logging_kibana_ca) | b64encode }}"
+  when: openshift_logging_kibana_ca | trim | length > 0
+  changed_when: false
+
+- set_fact:
+    kibana_ca: "{{ key_pairs | entry_from_named_pair('ca_file') }}"
+  when: kibana_ca is not defined
+  changed_when: false
+
+- name: Generating Kibana route template
+  template:
+    src: "route_reencrypt.j2"
+    dest: "{{ tempdir }}/templates/kibana-route.yaml"
+  vars:
+    obj_name: "{{ kibana_name }}"
+    route_host: "{{ openshift_logging_kibana_hostname }}"
```

Comment:
should this be the kibana ops hostname for the ops route?

Reply:
addressed here: https://github.com/openshift/openshift-ansible/pull/11817/files#diff-0857978473a58a6c671b308aece000d4R115-R118